R/statistic-histogram.R

```diff
@@ -2,6 +2,32 @@
 #' 
 #' Determine accuracy of histogram release, given epsilon and delta, for the differentially 
 #' private histogram release.
+#' 
+#' In differential privacy, "accuracy" is defined as the threshold value above which a given value 
+#' is "significantly different" from the expected value. Mathematically, this is written as:
+#' \deqn{\alpha = Pr[Y > a]}
+#' Where \eqn{\alpha} is the statistical significance level, \eqn{a} is the accuracy, and \eqn{Y} is a random 
+#' variable following the Laplace distribution.
+#' 
+#' In other words, the accuracy is a threshold value, above which a value is "unexpected". We call
```

Comment:
Maybe remove lines 12-14? I feel like it's an ok intuitive explanation of accuracy but that it might rub some stats people the wrong way. We can probably assume that given the technical definition above, people will have an ok intuitive understanding of accuracy.